### ISSUE TITLE
Show an obvious indicator that an updated extension requires a reload

### ISSUE THREAD

**Issue by bmewburn** [Microsoft/vscode#69620]
When an extension has auto-updated it displays a reload required notice on the extension inside the extensions view. 

This is only noticeable if you open the extensions view and on long running vscode sessions a user may not realise they should reload. This can result in users filing to duplicate and/or previously resolved error reports on the extension repo. Could a more obvious notification be shown?

![screenshot from 2019-03-01 09-18-51](https://user-images.githubusercontent.com/5885195/53602499-1c5df980-3c03-11e9-942e-0a976031334b.png)

**Comment by tmknight** [comment]
Agree.  There is a brief indicator on the Extension side-bar icon that updates are occurring or at least assessment for updates.  Just as with the Gear showing an update is pending, the Extension icon should show a reload pending.
